### PR TITLE
010 : Create project structure

### LINKED ISSUE #10: Create project structure
- Assets folder with subfolders img and CSS
- Create reset.css inside CSS folder (reset eric meyer)
- Create empty file style.css inside CSS folder
- Create a base.html (only with header, footer and import CSS, Bootstrap CDN and Fontawesome)

### PR DESCRIPTION
**Pull request by ledukilian**
close #10

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,135 @@
+<!doctype html>
+<html lang="fr">
+      <head>
+            <meta charset="UTF-8">
+            <!-- Balises meta, utilité ? -->
+            <meta name="description" content="Maquette du projet 3 Openclassrooms">
```
Comment (Tutt-J) [inline index.html:6]: Please put a real description

File: index.html [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+            <meta name="keywords" content="Maquette, Projet, Openclassrooms">
```
Comment (Tutt-J) [inline index.html:7]: Please put reals keywords

**Review by Tutt-J — CHANGES_REQUESTED** [review]
Please use Bootstrap CDN and put css files on assets folder

**Comment by ledukilian** [comment]
close #10

**Review by Tutt-J — CHANGES_REQUESTED** [review]
Please remove all class

**Review by Tutt-J — APPROVED** [review]

**Pull request merged**